fabtests/common/shared.c

```diff
@@ -563,7 +563,8 @@ int ft_alloc_msgs(void)
 		if (ret)
 			return ret;
 
-		ret = ft_alloc_host_tx_buf(MAX(tx_size, FT_MAX_CTRL_MSG) * opts.window_size);
+		size_t host_tx_size =  (opts.options & FT_OPT_ALLOC_MULT_MR) ? tx_mr_size : tx_size;
```

Comment:
can we move the declaration to the beginning of the function.

Reply:
Done.